src/Common/Sidebar.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Sidebar = () => {
+  return (
+        <div>
+            I am sidebar
+        </div>
```

Comment:
mixed indentation

Reply:
This is also fixed. Linting wasn't working correctly in this PR.